### ISSUE TITLE
Strange output for WRITE command

### ISSUE THREAD

**Issue by sofeikov** [Microsoft/CNTK#484]
Hi All,

I've been trying to run simple convolution network for MNIST dataset. I've converted the initial training set to set of pictures, but kept training and test sets separated as they were. The training process goes fine, the testing accuracy is 0.08%. However, when I try to run WRITE on the same test set command I do see strange output consisting mainly of 6s and 2s. 

What might be the reason?

Please, see my attached config and output files.

Any help is appreciated.
[02_Convolution.cntk.txt](https://github.com/Microsoft/CNTK/files/269161/02_Convolution.cntk.txt)
[02_Convolution.ndl.txt](https://github.com/Microsoft/CNTK/files/269160/02_Convolution.ndl.txt)
[PL.txt](https://github.com/Microsoft/CNTK/files/269162/PL.txt)

**Comment by Alexey-Kamenev** [comment]
Hello,
Please remove this section from your `write` command:
```
    NDLNetworkBuilder = [
        networkDescription = "$ConfigDir$/02_Convolution.ndl"
    ]

```
This is a known issue which hopefully will be fixed soon.

**Comment by sofeikov** [comment]
Hi Alexey,

thanks a lot! It did the trick

Regards,
Konstantin

P.S. To anyone who can come across this, you might have to retrain your model
